x-pack/metricbeat/module/aws/cloudwatch/_meta/fields.yml

```
@@ -0,0 +1,22 @@
+- name: cloudwatch
+  type: group
+  description: >
+    `cloudwatch` contains the metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by different namespaces.
+  release: beta
+  fields:
+    - name: cloudwatch.namespace
```

```suggestion
    - name: namespace
```

Prefix is already defined by the group name, you can remove it from here (check resulting documentation), same for the others

---

ha my bad, this is fixed! thanks!